crypto/test/Test/Pos/Crypto/CryptoSpec2.hs

```diff
@@ -1,6 +1,6 @@
 -- | Pos.Crypto specification
 
-module Test.Pos.CryptoSpec
+module Test.Pos.Crypto.CryptoSpec2
```

Comment:
Name this module `CryptoSpec2` because the `crypto` package already had a `CryptoSpec` package. Keeping these two separate for now to make reviewing easier, but these will be merged in the next PR.